src/web/scripts/entrypoint.sh

```diff
@@ -13,6 +13,18 @@ envsubst '\$HOST' < \
   /etc/nginx/nginx.template.conf > \
   /etc/nginx/nginx.conf
 
+
+# If SSL Certificate folder isn't present, generate one
+if [ ! -d "/etc/nginx/certificate" ];then
```

Comment:
Can also check if there are certificates in the folder? What do you think

Reply:
Oh, didn't think of that! Easy fix!